cmake/UraniumTranslationTools.cmake

```diff
@@ -108,6 +108,6 @@ if(GETTEXT_FOUND)
         TARGETS_FOR_PO_FILES(${language})
         TARGETS_FOR_MO_FILES(${language})
     endforeach()
-    install(DIRECTORY ${CMAKE_BINARY_DIR}/resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium/)
+    install(DIRECTORY ${CMAKE_BINARY_DIR}/resources DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/)
```

Comment:
Ok, let it unmerged then. I'll add a TODO somewhere and report it on JIRA, so we can plan there, when we will be able to improve the macro with an option.
When finished I'll merge it on my own :wink: 

Thanks!